cluster-config/cluster.yml

```diff
@@ -0,0 +1,218 @@
+apiVersion: kops/v1alpha2
```

Comment:
Did you mean to add all these files to the commit? This looks like the output from the pentagon migration run?

Reply:
Good catch, this should not be here.